components/create_post/create_post.tsx

```diff
@@ -590,6 +614,10 @@ class CreatePost extends React.PureComponent<Props, State> {
 
         this.props.actions.setDraft(StoragePrefixes.DRAFT + channelId, null);
         this.draftsForChannel[channelId] = null;
+
+        if (this.props.tutorialStep === Constants.TutorialSteps.POST_POPOVER && this.props.prewrittenMessages && this.props.prewrittenMessages !== PrewrittenMessagesTreatments.NONE) {
```

Comment:
nit: I think it is a good practice to make code self documented and put this kind of long conditions in a constant value to make code self explanatory and easy to read, for example, in this case, it could be something like `const shouldShowPrewrittenMsgs = this.props.tutoriaStep === Contants.Tuto....`